### DISCUSSION TITLE
fail2ban filter for coturn

### DISCUSSION THREAD

**Discussion by un99known99** [coturn/coturn#1379] [Q&A]
Hi,

are there suggestions, on how coturn can be protected by fail2ban?

I can see often entries like 
720: : session 001000000000000003: delete: realm=<turn.example.com>, username=<1705345356>
720: : session 001000000000000004: usage: realm=<turn.example.com>, username=<1705345356>, rp=5, rb=200, sp=5, sb=404
720: : session 001000000000000004: peer usage: realm=<turn.example.com>, username=<1705345356>, rp=0, rb=0, sp=0, sb=0
720: : session 001000000000000004: closed (2nd stage), user <1705345356> realm <turn.example.com> origin <>, local 0.0.0.0:8346, remote 177.5.226.23:1486, reason: allocation timeout
720: : session 001000000000000004: delete: realm=<turn.example.com>, username=<1705345356>

nagging my system.

Thanks in advance.

BR,
Mike

**Comment by eakraly** [comment]
Hi @un99known99 
This is not some kind of issue or someone attacking your system
A client created an allocation and then did not use it - so it is cleaned up after a timeout (600s by default)
You do not need to block those IPs - they are your clients!

**Comment by un99known99** [comment]
@eakraly 
Hi,
I do have some unfriendly port knockers like

52973: : IPv4. tcp or tls connected to: 91.148.190.158:35466
53033: : session 000000000000000001: TCP socket closed remotely 91.148.190.158:35466
53033: : session 000000000000000001: closed (2nd stage), user <> realm <turn.example.org> origin <>, local 0.0.0.0:9273, remote 91.148.190.158:35466, reason: TCP connection closed by client (callback)

which are not my calling clients.

Currently trying to get them out via fail2ban

Any other proposals, am I using the right traces in the logfile?
Any hint / suggestions how to deal with it?

**Comment by un99known99** [comment]
@eakraly 
Hi,
as of now I am filtering / blocking ips with entry:

"311963: : IPv4. tcp or tls connected to: 45.227.254.49:65158"

is that the appropriate way?
